packages/liquidator/src/liquidationStrategy.js

```diff
@@ -219,24 +204,24 @@ module.exports = (
   // Returns true if enough of the withdrawal liveness has passed that the WDF strategy can be activated.
   function passedDefenseActivationPercent({ position, currentBlockTime }) {
     return (
-      hasWithdrawRequestPending({ position, currentBlockTime }) &&
```

Comment:
I decoupled helper methods from each other, this way the function names make it more clear what they are returning